src/Containers/MiddleCanvas/Template.tsx

```diff
@@ -0,0 +1,159 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { 
+    DragDropContext, 
+    Droppable,
+    DroppableProvided,
+    Draggable,
+    DraggableProvided,
+    DraggableStateSnapshot,
+    DropResult,
+} from 'react-beautiful-dnd';
+import { ITemplatePrefill } from '/MiddleCanvasTypes';
```

Comment:
This should be './MiddleCanvasTypes' when changed I get a lot of 6 TS errors in this file.

Reply:
Ah, it is because I made a general type string[] | string[][] are not really the same. I will remove the general type and split into two types. Thanks